docs/csa/audit-and-compliance.md

```diff
@@ -0,0 +1,170 @@
+* [**HIPAA**](https://www.cdc.gov/phlp/publications/topic/hipaa.html). The Health Insurance Portability and Accountability Act, or HIPAA, is a U.S. federal law that requires “the creation of national standards to protect sensitive patient health information from being disclosed without the patient’s consent or knowledge.”
+* [**NIST**](https://www.ftc.gov/business-guidance/small-businesses/cybersecurity/nist-framework). The National Institute of Standards and Technology, or NIST, is responsible for multiple compliance frameworks related to cybersecurity and privacy controls. Similar to some HIPAA applications, adherence to the NIST framework is voluntary, but its implementation helps companies identify, protect, detect, respond and recover from cybersecurity threats and attacks. 
+* [**CMMC**](https://dodcio.defense.gov/CMMC/about/). The Cybersecurity Maturity Model Certification (CMMC) is an assessment framework developed in alignment with the U.S. Department of Defense (DoD)’s information security requirements. It generally applies to companies that are either DoD contractors or engage with the [Defense Industrial Base Sector](https://www.cisa.gov/topics/critical-infrastructure-security-and-resilience/critical-infrastructure-sectors/defense-industrial-base-sector) or DIB.
+* [**ISO 27001**](https://www.iso.org/standard/27001). The International Standards Organization (ISO) publishes various industrial and commercial standards. These standards, such as ISO 27001, are vital to world trade initiatives and work to establish common standards for organizations that must adhere to different countries’ requirements and priorities.
+* [**PCI-DSS**](https://www.pcisecuritystandards.org/). The Payment Card Industry Data Security Standard (PCI DSS) outlines best practices and drives “adoption of data security standards and resources for safe payments worldwide.” It is mandatory for any organization handling payment cardholder data. Failure to adhere to it can result in financial penalties.
+
+#### Security controls
+
+Security controls refer to internally-defined protective measures taken to prevent, detect, or mitigate security risks as they apply to physical assets, information, computer networks, or other assets, and can include: 
+* **Physical**. Controls that relate to physical facilities and assets, for example, restricting access to server rooms or other sensitive areas, or using surveillance cameras for real-time monitoring.
+* **Digital**. Controls that relate to basic network and application security protocols, like required formats for usernames and passwords, two-factor authentication, and up-to-date antivirus software.
+* **Cybersecurity**. Controls to prevent cyber attacks, or quickly mitigate them if they do occur or are suspected. Examples include data encryption and firewalls.
```

Comment:
```suggestion
* **Cybersecurity**. Controls to prevent or quickly mitigate cyber attacks if they do occur or are suspected. Examples include data encryption and firewalls.
```